soh/src/overlays/actors/ovl_En_Tk/z_en_tk.c

```diff
@@ -9,6 +9,10 @@
 #include "objects/object_tk/object_tk.h"
 
 #define FLAGS (ACTOR_FLAG_0 | ACTOR_FLAG_3)
+#define COLLECTFLAG_GRAVEDIGGING_HEART_PIECE 0x19
+#define ITEMGETINFFLAG_GRAVEDIGGING_HEART_PIECE 0x1000
+
+bool heartPieceSpawned;
```

Comment:
This needs to be added to the save states. I'd also give it a more specific name so there are no clashes.

Reply:
Hmm, can I add that to save states? It's a local variable to that actor right? Or did I inadvertently make a global variable when I did that.